### ISSUE TITLE
TgtgPollingError: Please accept terms first, validate your email and then retry!

### ISSUE THREAD

**Issue by xixov** [ahivert/tgtg-python#126]
I am using the latest version (0.8.0) and when following the instructions (I tried both `client.get_credentials()` and `client.login()`) I get the following error:
```
./tgtg/__init__.py in login(self)
    127                 first_login_response = response.json()
    128                 if first_login_response["state"] == "TERMS":
--> 129                     raise TgtgPollingError(
    130                         "Please accept terms first, validate your email and then retry!"
    131                     )

TgtgPollingError: Please accept terms first, validate your email and then retry!
```
Everything is working fine with the app, I tried to log out, uninstall, etc. I get the mail from the app, never from the python script.

What is the way to accept the terms within tgtg-python?

**Comment by ahivert** [comment]
For the moment you have to login with this email on the app to accept terms. Once terms accepted it will be ok to use the script

**Comment by xixov** [comment]
I have used the app with this email, I even tried to uninstall and reinstall the app. I didn't get terms to accept though…

**Comment by ahivert** [comment]
After some tests, it looks like this is displayed when the email is not linked to a toogoodtogo account. I suspect a typo in your email.

Updated the code to display the email entered in the error message.
Updated the code to fix the signup by email from the code
New release available 0.10.0

Feel free to reopen the issue if it's not solve on your side
